gamemodes/terrortown/entities/weapons/weapon_ttt_wtester.lua

```diff
@@ -497,11 +470,18 @@ if SERVER then
     -- @hook
     -- @realm server
     function GAMEMODE:TTTFoundDNA(finder, toucher, ent) end
-else -- CLIENT
+end
```

Comment:
we didn't do that anywhere else before. But I like that, I think we should start doing this, because these big blocks are sometimes hard to understand. I'll merge this and we should make this a style rule for ourself.